Tests/app/fixtures/MagentoTestBundle/EventListener/ImportFinishEventListener.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\MagentoConnectorBundle\Tests\app\fixtures\MagentoTestBundle\EventListener;
+
+use Elasticsearch\Common\Exceptions\BadRequest400Exception;
+use ONGR\ConnectionsBundle\EventListener\ImportFinishEventListener as ParentListener;
+
+/**
+ * ImportFinishEventListener class.
+ */
+class ImportFinishEventListener extends ParentListener
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function onFinish()
+    {
+        try {
+            parent::onFinish();
+        } catch (BadRequest400Exception $e) {
+            // Ignore Empty bulk requests.
```

Comment:
this is ... intriguing. Is it really the best way to handle empty bulk requests? Under what circumstances of import you wouldn't have a single query to execute?